crates/locutus-runtime/tests/contract_interface.rs

```diff
@@ -24,12 +24,14 @@ fn test_contract(contract_path: &str) -> WrappedContract {
 
 fn get_guest_test_contract() -> RuntimeResult<(ContractStore, ContractKey)> {
     let mut store = ContractStore::new(test_dir(), 10_000);
+    // FIXME: Generate required test contract
```

Comment:
open an issue for this when you can please

Reply:
Created issue #276